src/views/NowPlayingView.vue

```diff
@@ -2,17 +2,20 @@
   <div>
     <div class="title text-center text-h4 font-weight-bold green--text text--darken-3">{{titleForMoviePage}}</div>
```

Comment:
this component is a main page.
MainPage should be a MainPage

Please, rename this component to `MainView.vue`

Reply:
Done